### ISSUE TITLE
https://www.gitbook.com/ 

### ISSUE THREAD

**Issue by vdimarco** [mtldata/aifest#6]
?

**Comment by pcuci** [comment]
@vdimarco yes, let's do it - I can't set it up myself, needs the `mtldata` webhook be put in place first

toyed with a fork, it's good, easy learning curve, and keeps all commit history - so yes, let's
